src/components/feed/NodeDetails/NodeDetails.tsx

```diff
@@ -352,7 +352,7 @@ function getCommand(
     }
   }
 
-  let command = `$> docker run --rm -v $(pwd)/in:/incoming -v $(pwd)/out:/outgoing \\\n${dock_image} ${selfexec} `;
+  let command = `$> apptainer exec --rm -v $(pwd)/in:/incoming -v $(pwd)/out:/outgoing \\\n${dock_image} ${selfexec} `;
```

Comment:
There are more differences between the usages of `docker run` and `apptainer exec` which have not been changed here. For instance, if I were to try running the command you provided, here is what I get:

```
Error for command "exec": unknown flag: --rm
```

Please re-review the documentation for `docker run` and `apptainer exec` and then fix the command accordingly. Let me know if you need help with that.

Reply:
I had a doubt. How can I pause the delete command while the processes are being executed. Apptainer doesn't have a command similar to --rm (as far as I could see), so how can I run `${selfexec}` and other commands before delete or is Apptainer procedural and handle that on its own?

Reply:
And I have 2 commands, one that would replace --rm and one for -v. Do I need to make them into one (I tried but I keep getting errors) or two would work?

Reply:
> is Apptainer procedural [...]

I do not understand what you mean by that.

`--bind` is the appropriate equivalent of `-v`,

Reply:
Yes, I have made the changes in the code accordingly.

Reply:
Background: a container is not the same thing as an image (a.k.a. container image, docker image). A container is an instance of an image.

A difference between Docker and Apptainer is that, in Docker, the following container states are valid:

- created but not started
- created and finished

In Apptainer, these two states do not exist. Apptainer containers must be created AND running. If containers are not running, then they do not exist.

With the `docker run` command, the `--rm` flag is necessary because otherwise upon finishing the container will enter the "created and finished" state. The desired final state is for the container to have finished and no longer exist.

Since Apptainer containers disappear as soon as they're finished, there is no equivalent to the `--rm` option (it happens automatically).